canonical-web-and-design/tutorials.ubuntu.com
# Combine renovate updates

## Done

Combined the following renovate updates:
- https://github.com/canonical-websites/tutorials.ubuntu.com/pull/924
- https://github.com/canonical-websites/tutorials.ubuntu.com/pull/925
- https://github.com/canonical-websites/tutorials.ubuntu.com/pull/926

The required tests for each of these branches will never pass individually because they are all dependent on each other.

Also added new eslint rule as we can't safely use `const` without adding transpilation to this project which is a bigger task than updating some dependencies.

Also added some warnings to be ignored in polymer config as it fails otherwise.


## QA

- Check out this feature branch
- Run `yarn`
- Run `bower install` (you may need to install bower)
- Run `yarn test`
- Everything should pass
